### PR TITLE
feat(audio): batch  5 of native updates

### PR DESCRIPTION
**Pull request by AvarianKnight**

### CODE REVIEW

File: AUDIO/HasLoadedMpDataSet.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+---
+## HAS_LOADED_MP_DATA_SET
+
+```c
+// 0x544810ED9DB6BBE6
+BOOL HAS_LOADED_MP_DATA_SET();
+```
+
+## Return value
+Whether the audio has loaded the MP data set
```
Comment (PsychoShock) [inline AUDIO/HasLoadedMpDataSet.md:13]: `Return a bool whether the audio has loaded the MP data set`

File: AUDIO/HasLoadedSpDataSet.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+## HAS_LOADED_SP_DATA_SET
+
+```c
+// 0x5B50ABB1FE3746F4
+BOOL HAS_LOADED_SP_DATA_SET();
+```
+
+
+## Return value
+Whether the audio has loaded the SP data set
```
Comment (PsychoShock) [inline AUDIO/HasLoadedSpDataSet.md:14]: `Return a bool whether the audio has loaded the SP data set`

File: AUDIO/HintAmbientAudioBank.md [hunk 3]
```diff
@@ -5,15 +5,13 @@ ns: AUDIO
 
 ```c
 // 0x8F8C0E370AE62F5C 0xF1850DDC
-Any HINT_AMBIENT_AUDIO_BANK(Any p0, Any p1);
+cs_type(Any) BOOL HINT_AMBIENT_AUDIO_BANK(cs_type(Any) char* bankName, cs_type(Any) bool bOverNetwork);
```
Comment (PsychoShock) [inline AUDIO/HintAmbientAudioBank.md:8]: This native has 3 param actually.
- The bank name
- If network
- Player bits (int)

File: AUDIO/HintMissionAudioBank.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+aliases: ["0x40763EA7B9B783E7"]
+---
+## HINT_MISSION_AUDIO_BANK
+
+```c
+// 0x40763EA7B9B783E7
+cs_type(Any) BOOL HINT_MISSION_AUDIO_BANK(char* bankName, cs_type(int) BOOL bOverNetwork, int playerBits);
+```
+
+This is marked as a deprecated function internally, please use [HINT_SCRIPT_AUDIO_BANK](#_0xFB380A29641EC31A) instead.
```
Comment (PsychoShock) [inline AUDIO/HintMissionAudioBank.md:12]: Where did you get this? "marked as a deprecated function internally".

This native works correctly, it just dont guarentee the loading of the bank name, that's why `HINT_SCRIPT_AUDIO_BANK` is a better use.

File: AUDIO/HintScriptAudioBank.md [hunk 5]
```diff
@@ -5,15 +5,17 @@ ns: AUDIO
 
 ```c
 // 0xFB380A29641EC31A 0x41FA0E51
-Any HINT_SCRIPT_AUDIO_BANK(Any p0, int p1);
+cs_type(Any) BOOL HINT_SCRIPT_AUDIO_BANK(cs_type(Any) char* bankName, cs_type(int) BOOL bOverNetwork);
```
Comment (PsychoShock) [inline AUDIO/HintScriptAudioBank.md:8]: This native has 3 param actually:

- The bank name
- If network
- Player bits (int)

File: AUDIO/HintAmbientAudioBank.md (hunk 3 above)
Comment (AvarianKnight) [reply AUDIO/HintAmbientAudioBank.md:8]: This doesn't matter, this native shouldn't be used, and you can't add additional arguments to a native doc without breaking compatibility.

File: AUDIO/HintMissionAudioBank.md (hunk 4 above)
Comment (AvarianKnight) [reply AUDIO/HintMissionAudioBank.md:12]: That is how the natives are marked in the docs.

File: AUDIO/HintScriptAudioBank.md (hunk 5 above)
Comment (AvarianKnight) [reply AUDIO/HintScriptAudioBank.md:8]: You can't add native arguments as it will break the comparability of the type.

**Review by 4mmonium — APPROVED** [review]
LGTM, merging, thanks!

**Pull request merged**